README.rdoc

```diff
@@ -205,64 +205,52 @@ You kick butt. You've got your code reviewed and now you're ready to merge it do
-You'll be presented with a prefilled commit message based on the body of your pull request which includes the text <code>Closes #XX</code> that will automatically close the associated pull request on github when deliver completes.
+You'll be presented with a prefilled commit message based on the body of your pull request with references to the pull request and reviewers.
 
-Make a mistake and deliver too early? It happens. You'll be prompted after you edit your commit message if you want to push your updated +master+ to github. If you answer 'n', then you'll want to do <code>git reset --hard origin/master</code> and checkout your branch again.
+Want to clean up your feature branch afterwards? You'll be prompted after you edit your commit message if you want to clean up your +feature_branch+ on github. If you answer 'n', then your feature_branch will exist for you to clean it up later.
 
 This is what it looks like:
+
   $ git reflow deliver
-  From github.com:reenhanced/gitreflow
-  * branch            master     -> FETCH_HEAD
-  Merging pull request #36: 'Enforce at least one LGTM before delivery', from 'reenhanced:nh-fail-without-lgtm' into 'reenhanced:master'
+  Here's the status of your review:
+      branches:     simonzhu24:test1234 -> simonzhu24:master
+      number:       51
+      reviewed by:
+      url:          https://github.com/simonzhu24/test/pull/51
+
+  [notice] No one has reviewed your pull request.
+  Would you like to open it in your browser? n
+  This is the current status of your Pull Request. Are you sure you want to deliver? Y
+  Merging pull request #51: 'last commit message', from 'simonzhu24:test1234' into 'simonzhu24:master'
+  git checkout master
   Switched to branch 'master'
-  From github.com:reenhanced/gitreflow
+  Your branch is ahead of 'origin/master' by 1 commit.
+    (use "git push" to publish your local commits)
+
+  [success] Pull Request successfully merged.
+  Would you like to cleanup your feature branch? Y
+  git pull origin master
+  remote: Counting objects: 1, done.
+  remote: Total 1 (delta 0), reused 0 (delta 0), pack-reused 0
+  Unpacking objects: 100% (1/1), done.
+  From https://github.com/simonzhu24/test
    * branch            master     -> FETCH_HEAD
-  Already up-to-date.
-  Switched to branch 'nh-fail-without-lgtm'
-  Switched to branch 'master'
-  Updating c2ec1b1..f90e111
+     0d8f5e0..f853efa  master     -> origin/master
+  Updating 0b6782f..f853efa
   Fast-forward
-  Squash commit -- not updating HEAD
-   lib/git_reflow.rb | 71 +++++++++++++++++++++++++++----------------------------
-   1 file changed, 35 insertions(+), 36 deletions(-)
-  [master d1b4dd5] Enforces LGTM before deliver, even with no comments.
-   1 file changed, 35 insertions(+), 36 deletions(-)
-  Merge complete!
-  Would you like to push this branch to your remote repo and cleanup your feature branch? y
-  Counting objects: 7, done.
-  Delta compression using up to 16 threads.
-  Compressing objects: 100% (4/4), done.
```

Comment:
@simonzhu24 We should not make the default process fail.  If it fails because you are "currently on the branch" then perhaps we should switch to another branch to allow the delete?

Also the example in the readme should be happy path, showing everything working.